### PR TITLE
responsiveness

### PR DESCRIPTION
**Pull request by sb39**
issue #65  resolved

### CODE REVIEW

File: src/Components/Footer/Footer.css [hunk 1]
```diff
@@ -18,4 +18,11 @@ a {
 
 .footer__social {
     width: 100%;
-}
\ No newline at end of file
+}
+.col-md-3 row social my-2 align-items-center d-flex justify-content-center{padding:0px; margin:0px;}
```
Comment (ezioda004) [inline src/Components/Footer/Footer.css]: Please revert these changes, the footer looked better before.

**Comment by ezioda004** [comment]
You seem to have made many unnecessary changes, for example.
1. The ` <section className="main-section">` looked fine the way it is. Revert that back
2. You have changed the image used in challenge section, please refer to this mockup and use the same image that was there before. Mockup image [here](https://raw.githubusercontent.com/parulbindal/quiz-app/master/src/assets/images/Welcome%20Page.png)
3. Match the alignments of the section as in the image above. The text section is now smaller than before.
4.
screenshot 73
The text for all section should be above the images.
5. Remove changes made in `Navbar.css`, `Footer.css`, it looks fine the way it was before.

This issue has been open for too long, so you have till evening, 8 o'clock to be precise to do necessary changes or this issue will be assigned to someone else.

File: src/Components/Navbar/Navbar.css [hunk 2]
```diff
@@ -50,3 +50,7 @@ li {
   }
 }
 
+.container-fluid ,.nav , .navbar-brand {
```
Comment (ezioda004) [inline src/Components/Navbar/Navbar.css]: Revert this too.

File: src/Components/Welcome Page/Welcome.css [hunk 3]
```diff
@@ -10,5 +10,7 @@
 #welcome{
   display:flex;
   flex-direction:column;
+
```
Comment (ezioda004) [inline src/Components/Welcome Page/Welcome.css]: Unnecessary indention added.

**Comment by sb39** [comment]
should i close this pull ?

**Comment by ezioda004** [comment]
@sb39 No, update this PR. Don't make new ones.

**Comment by sb39** [comment]
please verify it's done or not

**Comment by sb39** [comment]
@ezioda004 can you tell, what you mean by the text over image, is it like the first div or you mean to tell, the (first text block and then pic block) * 3 ?

**Comment by ezioda004** [comment]
@sb39 Yes, exactly. First text block and then image.

**Comment by sb39** [comment]
all issues have been resolved

**Comment by sb39** [comment]
layout updated according to mock
